spec/std/static_array_spec.cr

```diff
@@ -79,6 +79,20 @@ describe "StaticArray" do
     a.to_s.should eq("StaticArray[1, 2, 3]")
   end
 
+  it "fills" do
```

Comment:
Using method signatures is IMO preferable because it more clearly communicates what's tested.
```suggestion
  it "#fill" do
```
Next one could be `"fill(&)"`.